pkg/common/cns-lib/volume/manager.go

```diff
@@ -208,6 +210,14 @@ func (m *defaultManager) CreateVolume(ctx context.Context, spec *cnstypes.CnsVol
 	}
 	volumeOperationRes := taskResult.GetCnsVolumeOperationResult()
 	if volumeOperationRes.Fault != nil {
+		fault, ok := volumeOperationRes.Fault.Fault.(cnstypes.CnsAlreadyRegisteredFault)
+		if ok {
+			log.Info("Observed CnsAlreadyRegisteredFault fault")
+			log.Infof("CreateVolume: Volume created successfully. VolumeName: %q, opId: %q, volumeID: %q", spec.Name, taskInfo.ActivationId, fault.VolumeId.Id)
```

Comment:
Instead of 2 log lines, can we change this to:
<pre>
log.Infof("CreateVolume: Volume is already registered with CNS. VolumeName: %q, opId: %q, volumeID: %q", spec.Name, taskInfo.ActivationId, fault.VolumeId.Id)
</pre>

Reply:
fixed.